lib/adapters/REST/endpoints/entry.ts

```diff
@@ -178,3 +178,15 @@ export const createWithId: RestEndpoint<'Entry', 'createWithId'> = <
     }
   )
 }
+
+export const references: RestEndpoint<'Entry', 'references'> = (
+  http: AxiosInstance,
+  params: GetSpaceEnvironmentParams & { entryId: string; maxDepth: number }
```

Comment:
`maxDepth` should be declared optional if you provide a default value in line 186.